fieldservice_maintenance/models/fsm_order_type.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 - TODAY Marcel Savegnago - Escodoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class FsmOrderType(models.Model):
+
+    _inherit = "fsm.order.type"
+
+    internal_type = fields.Selection(
+        selection_add=[("maintenance", "Maintenance")],
```

Comment:
14.0 now requires ondelete= attribute for selection_add. Please see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

Reply:
Improved code. Thanks.